util/reggen/README.md

```diff
@@ -637,7 +637,19 @@ The individual bundled wires are associated with the fields rather than the full
 
 ### Error responses
 
-Writes and reads that target addresses that are not represented within the register list typically return an error.
+Writes and reads that target addresses that are not mapped within the register list may return an error.
+Whether an error is returned depends on the target address and the number of address bits used internally:
+* Accesses to target addresses that can be fully represented with the internally used address bits typically return an error.
```

Comment:
Maybe add "unmapped" before target addresses here (because we don't return an error when it's a good address...)